Automattic/wp-calypso
# Bundled Plugins: Add redirect to bundle plugin flow from the themes/thanks-modal

#### Proposed Changes

* In the Theme Showcase, after switching to a theme with a bundled software set, the thanks modal will send your browser to `/setup/?siteSlug=${ siteSlug }&flow=plugin-bundle`, to allow you to set up your store and possibly atomic transfer.

#### Testing Instructions

* Be in an environment where `'themes/plugin-bundling'` is on (like calypso localhost)
* Visit the theme showcase `http://calypso.localhost:3000/themes/YOURSITE`
* Change themes to baxter (which currently has a bundled software set)
* You should be redirected to the plugin bundle flow by the thank you modal

#### Followups

* Detect if someone has already set up a store. Imagine if I go 2020 theme -> Baxter -> set up store -> 2020 theme -> Baxter. I probably shouldn't go through the stepper plugin-bundle flow again.
* Permissions. What if someone doesn't have a business plan and somehow gets here? Ideally it shouldn't happen, but it should be planned for. Maybe it's the stepper detecting eligibility and kicking you out.
